fixes positioning of colorpicker BUT ALSO HAS A BUG

felixhayashi opened the pull request:
@AlexDM0 not sure but for some reason, once the colorpicker is open, the buttons won't work correctly and the huecircle jumps back.

However the PR fixes the positioning issue (see screenshots below).

@AlexDM0 could you take over at that point please?

![2016-01-04 18 51 11](https://cloud.githubusercontent.com/assets/4307137/12096376/3e4adb64-b315-11e5-8958-8653cf115322.png)
![2016-01-04 18 50 51](https://cloud.githubusercontent.com/assets/4307137/12096377/3e4fd74a-b315-11e5-813a-f39e72480195.png)

AlexDM0 commented:
Hi Felix, 

I'll see what i can do!

Thanks!

AlexDM0 commented:
Hi Felix,

Could you add an example of when it does not work? The example of the vis website works fine!

Cheers

AlexDM0 commented:
nevermind... rebuilding helps :)

AlexDM0 commented:
Hi Felix,

I've fixed the bug on the develop branch. The problem was that you insert the color picker into a div that has a click listener for said color picker :).

I've added an on close callback to restore the onclick method and removed it on showing the color picker.

Thanks for the pull!

Cheers

felixhayashi commented:
> I've fixed the bug on the develop branch. The problem was that you insert the color picker into a div that has a click listener for said color picker :).  I've added an on close callback to restore the onclick method and removed it on showing the color picker.

Ahh that's what was going on. Thanks for taking care of it!

-Felix
